build.gradle

```diff
@@ -42,7 +42,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '11'
+    String javaFxVersion = '11.0.1'
```

Comment:
WOW this has been bugging me for the longest time, thanks for finding and changing it